src/Silex/Provider/TranslationServiceProvider.php

```diff
@@ -53,6 +58,10 @@ public function register(Application $app)
             return new MessageSelector();
         });
 
+        $app['translator.cache_dir'] = null;
+        $app['translator.debug'] = function() use ($app) {
```

Comment:
this should use ``function ($app)`` to be consistent with the way the translator service is defined